pyiso/base.py

```diff
@@ -316,7 +316,8 @@ def unzip(self, content):
             # have zipfile
             z = zipfile.ZipFile(filecontent)
         except zipfile.BadZipfile:
-            LOGGER.error('%s: unzip failure for content:\n%s' % (self.NAME, content))
+            LOGGER.error('%s: unzip failure for content beginning:\n%s' % (self.NAME, str(content)[0:100]))
+            LOGGER.info('%s: Faulty unzip content:\n%s' % (self.NAME, content))
```

Comment:
Why still log the full content at info level? Debug level seems more appropriate given this pull request's objectives.

Reply:
It might be worth rebasing off of master to ensure the whitespace issues are resolved before we merge this - also I agree it should be at debug level instead of info.